new-docs/Makefile

```diff
@@ -76,6 +80,14 @@ test: install
 clean:
 	ls -A public | grep -v 'favicon.ico' | xargs -I {} echo 'public/'{} | xargs rm -rf
 
+build-search:
+	cd search && go mod vendor && go build -o ../searcher
+
+require-algolia-vars: require-ALGOLIA_APPLICATION_ID require-ALGOLIA_ADMIN_API_KEY require-ALGOLIA_INDEX
+
+search: require-algolia-vars clean build-prod build-search
```

Comment:
Why is a clean required before building search?

Reply:
Good question - it's possibly NOT required.  Let me double check if it overwrites the JSON